include/mbedtls/x509_crt.h

```diff
@@ -308,7 +308,9 @@ int mbedtls_x509_crt_parse_der( mbedtls_x509_crt *chain,
  *
  *                 Callbacks of this type are passed to and used by the
  *                 mbedtls_x509_crt_parse_der_with_ext_cb() routine when
- *                 it encounters an unsupported extension.
+ *                 it encounters either an unsupported extension or a
+ *                 "certificate policies" extension containing any
+ *                 unsupported certificate policies.
```

Comment:
There may be additional cases where we'd like to invoke the callback. If the design had to be tweaked once, there's a non-negligible chance that it'll need to be tweaked further. Therefore please add a note stating that future versions of the library may invoke the callback in other cases.

Reply:
Done in https://github.com/ARMmbed/mbedtls/pull/3419/commits/511bc8c57b2f1e89cce28440fa97cf49860646be